ios/engine/KMEI/KeymanEngine/resources/Keyman.bundle/Contents/Resources/keyboard.html

```diff
@@ -180,6 +180,25 @@
                 }
             }
         
+            function suggestionPopup(obj,custom,x,y,w,h) {
+              if(obj != null) {
+                var i;
+                var s = JSON.stringify(obj);
+                var shift = false;
+                var pos = x.toString() + ',' + y.toString();
+                var size = w.toString() + ',' + h.toString();
+                var frame = pos + ',' + size;
```

Comment:
So ... 4 parameters, not one. Saves multiple levels of parsing at the other end :)

Reply:
Either that, or bundle it all into the JSON!